orbit/cmd/orbit/orbit.go

```diff
@@ -51,6 +51,8 @@ import (
 	"gopkg.in/natefinch/lumberjack.v2"
 )
 
+const unusedFlagKeyword = "dummy"
```

Comment:
Can you add a comment to this variable saying that this is for supporting parameters on the installer (mainly because Windows uses cli flags instead of env vars)?

Reply:
Done